control/Controller.php

```diff
@@ -83,30 +145,59 @@ public function Link() {
 	}
 
 	/**
+	 * {@inheritdoc}
+	 *
+	 * Also set the URLParams
```

Comment:
`{@inheritdoc}` is probably not used properly here.

Please include `@param` and `@return` phpdoc.